packages/@aws-cdk/aws-apprunner/lib/vpc-connector.ts

```diff
@@ -0,0 +1,135 @@
+import * as ec2 from '@aws-cdk/aws-ec2';
+import * as cdk from '@aws-cdk/core';
+import { Construct } from 'constructs';
+import { CfnVpcConnector } from './apprunner.generated';
+
+/**
+ * Properties of the AppRunner VPC Connector
+ */
+export interface VpcConnectorProps {
+  /**
+   * A list of IDs of security groups that App Runner should use for access to AWS resources under the specified subnets.
+   *
+   * @default - the default security group of the VPC which allows all outbound traffic.
+   */
+  readonly securityGroups?: ec2.ISecurityGroup[];
+
+  /**
+    * A list of subnets that App Runner should use when it associates the service with a custom Amazon VPC.
+    */
+  readonly subnets: ec2.ISubnet[];
```

Comment:
Even though the L1 doesn't take a vpc property, I wonder if it would be better to model this how we do other L2s and do something like:

```ts
export interface VpcConnectorProps {
  readonly vpc: ec2.IVpc;
  readonly vpcSubnets?: ec2.SubnetSelection;
}
```
and then 
```ts
subnets: props.vpc.selectSubnets(props.vpcSubnets).subnetIds,
```

Reply:
This sounds good to me; updated accordingly. Additionally, per your other comment, we will need access to the VPC to create a new security group if none passed.